praktikumsplaner-backend/src/main/resources/db/migration/V8__Add_relation_from_Praktikumsstelle_to_Meldezeitrum.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE AUSBILDUNGSPRAKTIKUMSSTELLE ADD COLUMN meldezeitraumID varchar(36) not null default null;
+ALTER TABLE STUDIUMSPRAKTIKUMSSTELLE ADD COLUMN meldezeitraumID varchar(36) not null default null;
```

Comment:
that will not work. You define `not null` and `null` as default. What is your solution with the existing `stellen`.

Reply:
default value is set to 0 now. The existing entries will have to be edited, if they are going to be used